src/main/java/core/mate/academy/model/ExcavatorProducer.java

```diff
@@ -0,0 +1,11 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
```

Comment:
Don't begin class or method implementation with an empty line. It's good practice to start the class definition or method definition immediately after the class or interface declaration.